src/schemas/Comparison.json

```diff
@@ -33,6 +33,23 @@
       "required": ["comparison", "value"],
       "additionalProperties": false
     },
+    {
+      "properties": {
+        "comparison": {
+          "type": "string",
+          "enum": ["matches"]
+        },
+        "value": {
+          "type": ["string"]
+        },
+        "target": {
```

Comment:
Probably best not to support target attributes for regex. I'm not sure what use case this would ever be used for, and it opens up more redos opportunities, e.g.:

```
'resource.path': {comparison: 'matches', target: 'resource.headers.BadGuy'}
```

which would use the value of the BadGuy header as the regular expression.